src/site/markdown/model_formats.md

```diff
@@ -0,0 +1,1996 @@
+    <td>Object</td>
+    <td>N / A</td>
+    <td>true</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aS_vw">vw</a></td>
+    <td>Object</td>
+    <td>N / A</td>
+    <td>true</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_namespaces">namespaces</a></td>
+    <td>Object</td>
+    <td>N / A</td>
+    <td>true</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_numMissingThreshold">numMissingThreshold</a></td>
+    <td>Number</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_notes">notes</a></td>
+    <td>Array</td>
+    <td>String</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_spline">spline</a></td>
+    <td>Object</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+</table> 
+
+### (VW) JSON Field Descriptions
+
+#### (VW) modelType
+
+`modelType` field must be `VwJNI`.
+
+#### (VW) features
+
+See [Regression model features](#aR_features) for details.
+
+#### (VW) vw
+
+The `vw` Object has two main components.  The first component is a representation of the binary VW model, or 
+"*initial regressor*" in VW terminology.  This is the model parameterization.  The second component is set of 
+parameters used by VW.  This includes things like link function, prediction ranges, skip grams, namespace 
+interactions, regularization, etc.  To see descriptions of these parameters, see the 
+[VW wiki](https://github.com/JohnLangford/vowpal_wabbit/wiki) or run: 
+  
+```bash
+vw -h
+```
+
+<table>
+  <tr>
+    <th>Field Name</th>
+    <th>JSON Type</th>
+    <th>JSON Subtype</th>
+    <th>Required</th>
+    <th>Default</th>
+  </tr>
+  <tr>
+    <td><a href="#aVW_vw_creationTime">creationTime</a></td>
+    <td>Number</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>number of milliseconds since <a href="https://en.wikipedia.org/wiki/Unix_time">epoch</a> for current time.</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_vw_model">model</a></td>
+    <td>String</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_vw_modelUrl">modelUrl</a></td>
+    <td>String</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_vw_via">via</a></td>
+    <td>String with one of: "<em>file</em>", "<em>vfs1</em>", "<em>vfs2</em>"</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>N / A</td>
+  </tr>
+  <tr>
+    <td><a href="#aVW_vw_params">params</a></td>
+    <td>String or Array of String</td>
+    <td>N / A</td>
+    <td>false</td>
+    <td>Empty String</td>
+  </tr>
+</table>
+
+##### (VW) vw creationTime
+
+`creationTime` is optional and is used when copying the content of the binary VW model to the local disk on the 
+computer where the Aloha model is run.  When no value is supplied, the value will be set to
+`java.lang.System.currentTimeMillis()`.
+
+##### (VW) vw model
+
```

Comment:
This would be a great place to link to the docs about the CLI tool you wrote for VW model creation.